serpentTools/tests/test_ResultsReader.py

```diff
@@ -0,0 +1,233 @@
+"""Test the results reader."""
+
+import os
+import unittest
+
+import numpy
+
+import six
+
+from serpentTools.settings import rc
+from serpentTools.tests import TEST_ROOT
+from serpentTools.parsers import ResultsReader
+from serpentTools.messages import SerpentToolsException
+
+class _ResultsTesterHelperFull(unittest.TestCase):
+    """Test case to share setUpClass for testing Results Reader"""
```

Comment:
Change the docstrings and names of these helpers to reflect their purpose and scope

Reply:
"""
    Class with common tests for the results reader.

    Expected failures/errors:

        1. test_varsMatchSettings:
                        compares the keys
                        defined by the user to those
                        obtained by the reader
            Raises SerpentToolsException
        2. test_metadata:
                        Check that metadata variables and
                        their values are properly stored
            Raises SerpentToolsException
        3. test_resdata:
                        Check that time-dependent results variables
                        and their values are properly stored
            Raises SerpentToolsException
        4. test_univdata:
                        Check that expected states are read
                        i.e., ('univ', bu, buIdx, days)
                        For a single state, check that
                        infExp keys and values are stored.
                        Check that infUnc and metadata are properly stored
            Raises SerpentToolsException
    """